CHANGELOG.md

```diff
@@ -1,16 +1,28 @@
 # Changelog
 
 ## [Unreleased]
+
 * Changes
-  * Removed the `get_user_profile` method and the `email_address` and `profile` attributes from the `Auth` class ([#421](https://github.com/nsidc/earthaccess/issues/421))
-* Bug fixes:
-    * fixed 483 by extracting a common CMR query method for collections and granules using SearchAfter header
-    * Added VCR support for verifying the API call to CMR and the parsing of returned results without relying on CMR availability post development
 
-* Enhancements:
-  * Corrected and enhanced static type hints for functions and methods that make
-    CMR queries or handle CMR query results ([#508](https://github.com/nsidc/earthaccess/issues/508))
-  * Enable queries to Earthdata User Acceptance Testing (UAT) system for authenticated accounts ([#421](https://github.com/nsidc/earthaccess/issues/421))
+  * [#421](https://github.com/nsidc/earthaccess/issues/421): Removed the
+    `get_user_profile` method and the `email_address` and `profile` attributes
+    from the `Auth` class.  Calling the EDL API to get user profile information
+    is not intended for library access and is not necessary for this library's
+    intended use cases.
+  * [#555](https://github.com/nsidc/earthaccess/issues/555): YAML formatting is
+    now performed with `yamlfmt` instead of `prettier`.
+
+* Enhancements
+
+  * [#483](https://github.com/nsidc/earthaccess/issues/483): Now using
+    [Search After](https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html#search-after)
+    for collection and granule searches to support deep-paging through large
+    result sets.
+  * [#508](https://github.com/nsidc/earthaccess/issues/508): Corrected and
+    enhanced static type hints for functions and methods that make CMR queries
+    or handle CMR query results.
+  * [#421](https://github.com/nsidc/earthaccess/issues/421): Enabled queries to
+    Earthdata User Acceptance Testing (UAT) system for authenticated accounts.
```

Comment:
:star_struck: thank you :sparkles: 

I wonder if there's a changelog linting tool that can help us enforce a standard like the one you're setting here.